rir_delay.py

```diff
@@ -19,14 +19,6 @@ def spherical_distance(point1, point2):
 
     return d
 
-rir_delay.spherical_distance(m1_coords, m2_coords)
```

Comment:
@rupalisarathy I moved all of this code into the `if __name__ == '__main__':` block so it won't run when we import this module for it's functions